docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_DropdownMenu.htm

```diff
@@ -47,10 +47,11 @@ <h2><a id='manual-constructor0.6513923294842243'></a>Manual constructor</h2>
    &lt;option value="3"&gt;Item3&lt;/option&gt;
    &lt;option value="4"&gt;Item4&lt;/option&gt;
 &lt;/select&gt;
-&lt;script&gt;
-   var element = document.getElementById("dropdownmenu"),
-       widget = tau.widget.DropdownMenu(element);
-&lt;/script&gt;
+</pre>
+<p>JS code:</p>
```

Comment:
Global Suggestion : How about adding "<p>HTML code:</p>" as "<p>JS code:</p>" added?

If you want to keep current state, I'll merge this PR.